package.json

```diff
@@ -1,14 +1,15 @@
 {
-    "name": "OpenSlides",
-    "private": true,
-    "devDependencies": {
-        "bower": "~1.3.12",
-        "gulp": "~3.8.10",
-        "gulp-concat": "~2.4.3",
-        "gulp-if": "~1.2.5",
-        "gulp-minify-css": "~0.3.11",
-        "gulp-uglify": "~1.0.2",
-        "main-bower-files": "~2.4.1",
-        "yargs": "~1.3.3"
-    }
+  "name": "OpenSlides",
+  "private": true,
+  "devDependencies": {
+    "bower": "~1.3.12",
+    "gulp": "~3.8.10",
+    "gulp-concat": "~2.4.3",
+    "gulp-if": "~1.2.5",
+    "gulp-minify-css": "~0.3.11",
+    "gulp-uglify": "~1.0.2",
+    "main-bower-files": "~2.4.1",
+    "yargs": "~1.3.3",
+    "po2json": "~0.3.2"
```

Comment:
Please explain where we use the converted json files. They are not checked into the repo, are they? Maybe gulp should do the job.